package.json

```diff
@@ -15,12 +15,14 @@
     "setup": "prisma generate && prisma migrate deploy && prisma db seed",
     "start": "remix-serve build",
     "start:mocks": "binode --require ./mocks -- @remix-run/serve:remix-serve build",
-    "test": "vitest",
+    "test": "vitest run",
+    "test:watch": "vitest",
     "test:e2e:dev": "start-server-and-test dev http://localhost:3000 \"npx cypress open\"",
     "pretest:e2e:run": "npm run build",
     "test:e2e:run": "cross-env PORT=8811 start-server-and-test start:mocks http://localhost:8811 \"npx cypress run\"",
     "typecheck": "tsc -b && tsc -b cypress",
-    "validate": "run-p \"test -- --run\" lint typecheck test:e2e:run"
+    "validate": "run-p test lint typecheck test:e2e:run",
+    "validate:precommit": "run-p test lint typecheck"
```

Comment:
Totally with you on leaving off e2e testing for a pre-commit, that's a lot to ask.

Personally, I don't usually run tests on commits, but rather let CI/CD handle that. For this project though since w'ere using vitest and it's pretty small, it likely won't be a big deal.

Only command that might be nice is to add `format` in there as well, that way we can keep formatting consistent even for setups that don't have prettier auto-formatting